### PR TITLE
VZ-1315 zap structured logging

### PR DESCRIPTION
**Pull request by guoyongzhang**

### CODE REVIEW

File: THIRD_PARTY_LICENSES.txt [hunk 1]
```diff
@@ -9662,6 +9648,17 @@ github.com/rs/cors
 -------- Copyrights
 Copyright (c) 2014 Olivier Poitrey <rs@dailymotion.com>
 
+-------- Dependency
+github.com/rs/xid
```
Comment (markxnelson) [inline THIRD_PARTY_LICENSES.txt:9652]: unapproved dependency

File: cmd/manager/main.go [hunk 2]
```diff
@@ -180,3 +176,25 @@ func serveCRMetrics(cfg *rest.Config) error {
 	}
 	return nil
 }
+
+// InitLogs initializes logs with Time and Global Level of Logs set at Info
+func InitLogs() {
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+
+	// Log levels are outlined as follows:
```
Comment (markxnelson) [inline cmd/manager/main.go:184]: enum

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: THIRD_PARTY_LICENSES.txt (hunk 1 above)
Comment (shihchang) [reply THIRD_PARTY_LICENSES.txt]: updated  THIRD_PARTY_LICENSES.txt

File: cmd/manager/main.go (hunk 2 above)
Comment (shihchang) [reply cmd/manager/main.go]: fixed with zap config

File: cmd/manager/main.go [hunk 3]
```diff
@@ -40,49 +43,35 @@ var (
...
-
-	// Use a zap logr.Logger implementation. If none of the zap
-	// flags are configured (or if the zap flag set is not being
-	// used), this defaults to a production zap logger.
-	//
-	// The logger instantiated here can be changed to any logger
-	// implementing the logr.Logger interface. This logger will
-	// be propagated through the whole operator, generating
-	// uniform and structured logs.
-	logf.SetLogger(zap.Logger())
```
Comment (shihchang) [inline cmd/manager/main.go:72]: moved to line 204

**Review by shihchang — APPROVED** [review]

**Pull request merged**